setup.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
+
+####################################################################
+# ### setup.py                                                   ###
+####################################################################
+# ### Author: SAS Institute Inc.                                 ###
+####################################################################
+#                                                                ###
+# Copyright (c) 2021, SAS Institute Inc., Cary, NC, USA.         ###
+# All Rights Reserved.                                           ###
+# SPDX-License-Identifier: Apache-2.0                            ###
+#                                                                ###
+####################################################################
+
+from setuptools import find_packages, setup
+
+# get the install requirements from requirements.txt
+with open("requirements.txt") as requirements_file:
+    install_requirements = requirements_file.read().splitlines()
+
+setup(
+    # UPDATE VERSION WITH EACH RELEASE
+    ##################################
+    version="1.3.1",
```

Comment:
it would be better to source version from git tags instead, which I think is the default if not specified in this file. It looks like we're already tagging releases so that should be naturally picked up by current processes

https://github.com/sassoftware/viya4-ark/tags